src/Analysis/Engine/Impl/Values/SpecializedNamespace.cs

```diff
@@ -208,52 +209,17 @@ abstract class SpecializedNamespace : AnalysisValue {
             return _original.IsOfType(klass);
         }
 
-        public override IEnumerable<ILocationInfo> Locations {
-            get {
-                if (_original == null) {
-                    return new LocationInfo[0];
-                }
-                return _original.Locations;
-            }
-        }
+        public override IEnumerable<ILocationInfo> Locations => _original.Locations?.MaybeEnumerate();
 
-        public override string Name => _original == null ? base.Name : this._original.Name;
+        public override string Name => _original == null ? base.Name : _original.Name;
```

Comment:
Why in this case we call `base.Name`, and in previous explicitly return empty enumerable instead of calling `base.Locations`, which also returns empty enumerable?

Reply:
Good question. No idea.